src/shared/theme/useMediaQuery.tsx

```diff
@@ -0,0 +1,21 @@
+import { useEffect, useState } from 'react';
+
+export const useMediaQuery = (query: string) => {
+  const [matches, setMatches] = useState<boolean>(
+    window.matchMedia(query).matches,
+  );
+
+  useEffect(() => {
+    const matchQuery = window.matchMedia(query);
+    const onChange = (e: MediaQueryListEvent) => {
+      setMatches(e.matches);
+    };
+
+    matchQuery.addEventListener('change', onChange);
+    return () => matchQuery.removeEventListener('change', onChange);
+  }, [query]);
+
+  return matches;
+};
+
+
```

Comment:
should fix this warning as well. Looks like just an extra line at the end of the file

Reply:
> should fix this warning as well. Looks like just an extra line at the end of the file

It might be useful to have things like this cleaned up on commit via husky.